en/tools/config-catalog-generator-cc/data/adapter/configs.json

```diff
@@ -0,0 +1,200 @@
+{
+  "configs": [
+    {
+      "title": "Server",
+      "options": [
+        {
+          "name": "server",
+          "required": true,
+          "description": "This includes configurations required for configuring the deployment parameters that are used for identifying the Choreo Connect Adapter REST APIs.",
+          "params": [
+            {
+              "name": "enabled",
+              "type": "boolean",
+              "required": true,
+              "default": "true",
+              "possible": "true, false",
+              "description": "Enable/Disable Choreo Connect Adaptor REST API."
+            },
+            {
+              "name": "host",
+              "type": "string",
+              "required": true,
+              "description": "Hostname of the Choreo Connect Adaptor"
+            },
+            {
+              "name": "port",
+              "type": "string",
+              "required": true,
+              "default": "9843",
+              "description": "Port of the Choreo Connect Adaptor"
+            },
+            {
+              "name": "tokenTTL",
+              "type": "string",
+              "required": false,
+              "default": "1h",
+              "description": "Time to live for the Adaptor access token"
+            },
+            {
+              "name": "tokenPrivateKeyPath",
+              "type": "string",
+              "required": false,
+              "default": "/home/wso2/security/keystore/mg.key",
+              "description": "Private key path to use for the token generation"
+            }
+          ]
+        }
+      ],
+      "exampleFile": "server.toml"
+    },
+    {
+      "title": "Keystore",
+      "options": [
+        {
+          "name": "keystore",
+          "required": true,
+          "description": "The configurations of key store used in Choreo Connect Adapter",
+          "params": [
+            {
+              "name": "certPath",
+              "type": "string",
+              "required": true,
+              "default": "/home/wso2/security/keystore/mg.pem",
+              "description": "Path of the certificate of the Adaptor"
+            },
+            {
+              "name": "keyPath",
+              "type": "string",
+              "required": true,
+              "default": "/home/wso2/security/keystore/mg.key",
+              "description": "Path of the private key of the Adaptor"
+            }
+          ]
+        }
+      ],
+      "exampleFile": "keystore.toml"
+    },
+    {
+      "title": "Truststore",
+      "options": [
+        {
+          "name": "truststore",
+          "required": true,
+          "description": "The configurations of trust store used in in the Choreo Connect Adapter.",
```

Comment:
Typo in description -> in in
Also, the use of full stop at the end of descriptions is not consistent across the configs.json files. Isn't it better to maintain some sort of consistency?